sonic-psud/scripts/psud

```diff
@@ -38,6 +38,11 @@ PSU_INFO_TABLE = 'PSU_INFO'
 PSU_INFO_KEY_TEMPLATE = 'PSU {}'
 PSU_INFO_PRESENCE_FIELD = 'presence'
 PSU_INFO_STATUS_FIELD = 'status'
+PSU_INFO_TEMP_FIELD = 'temp'
+PSU_INFO_TEMP_TH_FIELD = 'temp_th'
+PSU_INFO_VOLTAGE_FIELD = 'voltage'
+PSU_INFO_VOLTAGE_MAX_TH_FIELD = 'voltage_max_th'
+PSU_INFO_VOLTAGE_MIN_TH_FIELD = 'voltage_min_th'
```

Comment:
Wondering if we should use the full word "threshold" in the threshold-related DB keys for clarity?

Reply:
Fixed